proto/process/agent.proto

```diff
@@ -843,3 +846,20 @@ message DNSStats {
 	uint64 dnsFailureLatencySum = 3;
 	map<uint32, uint32> dnsCountByRcode = 4;
 }
+
+message HTTPStats {
+	repeated Data statsByResponseStatus = 1;
+
+	message Data {
+		uint32 count = 1;
+		repeated bytes latencies = 2;
```

Comment:
Is this repeated because of the 4-part sketch format?

Reply:
Exactly - though I'm trying to see if we can just use `proto.Marshal` to serialize the data, and if that works I'll have to switch this back to just (not repeated) `bytes`

Reply:
I'm not sure that it will work since the field ordering is not supposed to be guaranteed in protobuf.  In practice it may be but that could change depending on the protobuf implementation / version